core/src/test/java/org/apache/brooklyn/core/mgmt/internal/EntityExecutionManagerTest.java

```diff
@@ -68,6 +68,7 @@
 import com.google.common.util.concurrent.Callables;
 
 /** Includes many tests for {@link BrooklynGarbageCollector} */
+@Test
```

Comment:
Agreed - seems unnecessary.

It's also a little confusing because some tests in this class are marked with `@Test(groups="Integration")`, which will (I believe) take precedence over the class' test annotation, but makes people pause and think.